### PR TITLE
Add French translations

### PR DESCRIPTION
**Pull request by regisrob**
... on the same model as @dicksonlaw583 for Chinese

### CODE REVIEW

File: src/i18n.js [hunk 1]
```diff
@@ -12,6 +13,7 @@ const resources = {
   en,
   'zh-CN': zhCn,
   'zh-TW': zhTw,
+  fr
```
Comment (camillevilla) [inline src/i18n.js]: Just two minor changes needed to make eslint happy and get the Travis build to pass. 
1) move this key above `'zh-CN'`
2) add a trailing comma after `fr`

Comment (regisrob) [reply src/i18n.js]: Ok, I will fix this asap (tomorrow at the latest)

Comment (regisrob) [reply src/i18n.js]: oh just saw it has already been done! Thanks @mejackreed

Comment (mejackreed) [reply src/i18n.js]: I just made the minor change as a new commit to this branch. I hope that is ok.

Comment (camillevilla) [reply src/i18n.js]: @regisrob! It turns out that maintainers can make edits to a PR from your fork. 
https://help.github.com/en/articles/allowing-changes-to-a-pull-request-branch-created-from-a-fork 

@mejackreed made the changes so we can ship this with today's release 🎉

**Pull request merged**